test/batchReceiver.spec.ts

```diff
@@ -304,35 +303,32 @@ describe("ReceiveBatch from Queue/Subscription", function(): void {
     should.equal(msgs[0].body, testMessages[0].body);
     should.equal(msgs[0].messageId, testMessages[0].messageId);
 
-    if (msgs[0].sequenceNumber) {
-      const sequenceNumber = msgs[0].sequenceNumber;
-      await msgs[0].defer();
+    if (!msgs[0].sequenceNumber) {
+      throw "The message that we are receving from the sequence number can not be null";
+    }
+    const sequenceNumber = msgs[0].sequenceNumber;
+    await msgs[0].defer();
 
-      const receivedMsgs = await queueClient.receiveBatch(1);
-      should.equal(Array.isArray(receivedMsgs), true);
-      should.equal(receivedMsgs.length, 1);
-      should.equal(receivedMsgs[0].body === testMessages[0].body, false);
-      should.equal(receivedMsgs[0].messageId === testMessages[0].messageId, false);
-      await receivedMsgs[0].complete();
-
-      const message = await queueClient.receiveDeferredMessage(sequenceNumber);
-      if (message) {
-        should.equal(message.body, testMessages[0].body);
-        should.equal(message.messageId, testMessages[0].messageId);
-
-        await message.complete();
-      } else {
-        throw "The message that we are receving from the sequence number can not be null";
-      }
-    } else {
-      throw "Sequence Number can not be null";
+    const receivedMsgs = await queueClient.receiveBatch(1);
+    should.equal(Array.isArray(receivedMsgs), true);
+    should.equal(receivedMsgs.length, 1);
+    should.equal(receivedMsgs[0].body === testMessages[0].body, false);
+    should.equal(receivedMsgs[0].messageId === testMessages[0].messageId, false);
+    await receivedMsgs[0].complete();
+
+    const deferredMsgs = await queueClient.receiveDeferredMessage(sequenceNumber);
+    if (!deferredMsgs) {
+      throw "The message that we are receving from the sequence number can not be null";
```

Comment:
Can we simplify this? How about "No deferred message received for sequence number" ?